feedbag/day1/1.expense_report.py

```diff
@@ -0,0 +1,211 @@
+import itertools
+
+# how do you pass external data into this code?
+#expense_report = [1721,979,366,299,675,1456] # using sample data for functional purposes
+expense_report = [
```

Comment:
Nothing wrong here.  If you wanted to dedupe, you could put your data in a file and use the python `file.open()` functions.